river/metrics/cluster/bic.py

```diff
@@ -0,0 +1,150 @@
+import math
+
+from river import metrics
+
+from . import base
+
+
+class BIC(base.InternalMetric):
+    r"""Bayesian Information Criterion (BIC).
+
+    In statistics, the Bayesian Information Criterion (BIC) [^1], or Schwarz Information
+    Criterion (SIC), is a criterion for model selection among a finite set of models;
+    the model with the lowest BIC is preferred. It is based, in part, on  the likelihood
+    function and is closely related to the Akaike Information Criterion (AIC).
```

Comment:
Does the clustering BIC differ from the traditional one? In other words, although the results are always negative, we are seemingly looking for the highest ones, right? Maybe the interpretation is what changed. Could you please elaborate on that, @hoanganhngo610?

Reply:
You are correct. The BIC here is not actually different from the traditional ones. I have also stated below that although the values are always negative, we are still looking for the highest ones.